app/src/config.ts

```diff
@@ -7,6 +7,8 @@ export const IS_PROD = process.env.NODE_ENV === 'production';
 // flag to check if the app is running in a a test environment
 export const IS_TEST = process.env.NODE_ENV === 'test';
 
+export const PUBLIC_URL = process.env.PUBLIC_URL;
```

Comment:
To avoid confusion, it would be a good idea to sanitize the value of `PUBLIC_URL` provided by the user. It would be very common to omit or add an unnecessary slash (ex: `/lit/`, `/lit`, or `lit/`). Specifying the wrong one will likely result in a blank page with a cryptic console error message. 

Since you are using paths like `${PUBLIC_URL}/settings` for the routes, I suggest we ensure that `PUBLIC_URL` always has a leading slash and no trailing slash after it's sanitized.

Reply:
That's already done by the framework. You can test it with different PUBLIC_URL settings, it'll remove the trailing slash if it is present.